python/test/t_MaximumLikelihoodFactory_std.py

```diff
@@ -15,3 +15,45 @@
 inf_distribution = factory.build(sample)
 print(inf_distribution)
 
+class TruncatedFactory(object):
+    def __init__(self, distribution, lb=None, ub=None):
```

Comment:
@regislebrun after more testing it seems optimizer have trouble getting the bounds rights, so I forget TruncatedDistributionFactory for now.